### DISCUSSION TITLE
Portable location

### DISCUSSION THREAD

**Discussion by mblennegard** [Flow-Launcher/Flow.Launcher#2507] [Q&A]
I have activated Portable mode and would like to save my profile to OneDrive to sync it between computers, but I cannot find any way to actually change the location, which to me kind of defeats the purpose of portable.

Am I just missing something in the Flow Launcher Settings?

**Comment by Garulf** [comment]
Once portable mode is active you're free to move the FlowLauncher folder wherever you'd like, including OneDrive.

**Reply by mblennegard** [reply]
Oh, allright. 
I moved the folder %localappdata%\FlowLauncher to my OneDrive folder and that seems to work, at least after the inital sync had completed (meanwhile all settings were gone, but they repopulated after a the sync completed and I restarted Flow Launcher).

It would be nice if it was possible to install Flow Launcher but to specify a separate (= portable) location for settings and plugins only.
I guess something like this could be achieved with some scripts to sync the relevant files, but that is always a bit error prone in my experience.

**Reply by Garulf** [reply]
This may be doable with symbolic links. Depending on your cloud drive's capabilities.
